Kickstarter-iOS/Views/Controllers/ProjectNavigatorViewController.swift

```diff
@@ -63,8 +63,6 @@ internal final class ProjectNavigatorViewController: UIPageViewController {
     )
     self.navigatorDelegate = navigatorDelegate
 
-    self.viewModel.inputs.configureWith(project: initialProject, refTag: refTag)
```

Comment:
Is `configureWith` only being used for analytics?

Reply:
Yes.